box-share-sdk/src/main/java/com/box/androidsdk/share/fragments/InviteCollaboratorsFragment.java

```diff
@@ -205,6 +226,89 @@ private void fetchInvitees() {
+            new BoxFutureTask.OnCompletedListener<BoxIteratorCollaborations>() {
+                @Override
+                public void onCompleted(final BoxResponse<BoxIteratorCollaborations> response) {
+                    dismissSpinner();
+                    final Activity activity = getActivity();
+                    if (activity == null) {
+                        return;
+                    }
+                    activity.runOnUiThread(new Runnable() {
+                        @Override
+                        public void run() {
+                            if (response.isSuccess() && getFolder() != null) {
+                                mCollaborations = response.getResult();
+                                updateUi();
+                            } else {
+                                BoxLogUtils.e(CollaborationsFragment.class.getName(), "Fetch Collaborators request failed",
+                                        response.getException());
+                                mController.showToast(getActivity(), getString(R.string.box_sharesdk_network_error));
+                            }
+                        }
+                    });
+                }
+            };
+
+    private void updateUi() {
+        if (mCollaborations != null && mCollaborations.size() != 0) {
+
+            mInitialsListHeader.setVisibility(View.VISIBLE);
+            final int totalCollaborators = mCollaborations.fullSize().intValue();
+            final int remainingWidth = mInitialsListView.getWidth();
+            final ArrayList<BoxCollaboration> collaborations = mCollaborations.getEntries();
+            //Add the first item to calculate the width
+            final View initialsView = AddInitialsToList(collaborations.get(0).getAccessibleBy());
+            initialsView.post(new Runnable() {
+                @Override
+                public void run() {
+                    int viewWidth = initialsView.getWidth();
+                    int viewsCount = remainingWidth/viewWidth;
+                    for (int i = 1; i < viewsCount && i < collaborations.size(); i++) {
+                        View viewAdded = AddInitialsToList(collaborations.get(i).getAccessibleBy());
+                        if (i == viewsCount - 1) {
+                            // This is the last one, display count if needed
+                            int remaining = totalCollaborators - viewsCount;
+                            if (remaining > 0) {
+                                TextView initialsTextView = (TextView) viewAdded.findViewById(R.id.collaborator_initials);
+                                CollaborationUtils.setInitialsThumb(getActivity(), initialsTextView, remaining + 1);
+                            }
+                        }
+                    }
+                }
+            });
+        }
+    }
+
+    private View AddInitialsToList(BoxCollaborator collaborator) {
```

Comment:
Can you make this lower case? Methods I think should be lower case while constructors and classes would be capital.